dragonfly/block/break_info.go

```diff
@@ -89,11 +89,18 @@ var axeEffective = func(t tool.Tool) bool {
 	return t.ToolType() == tool.TypeAxe
 }
 
-// shovelEffective is a convenience function for blocks that are effectively mined with an axe.
+// shovelEffective is a convenience function for blocks that are effectively mined with a shovel.
 var shovelEffective = func(t tool.Tool) bool {
 	return t.ToolType() == tool.TypeShovel
 }
 
+// hoeEffective is a convenience func tion for blocks that are effectively mined with a hoe.
+var hoeEffective = func(t tool.Tool) bool {
+	return t.ToolType() == tool.TypeHoe
+}
+
+//TODO: Add shearEffective
```

Comment:
You can remove the TODO here. These functions are only convenience functions - hence the documentation - and do not actually need to be there.